templates/zerver/emails/organization_info.txt

```diff
@@ -0,0 +1,21 @@
+{% extends "zerver/emails/compiled/email_base_default.html" %}
+
+{% block illustration %}
+<img src="{{ email_images_base_uri }}/email_logo.png" alt=""/>
+{% endblock %}
+
+{% block content %}
+{{ _('New organization details') }}
+
+{% if realm_creation %}
+    {% trans %} Organization Name : <b>{{ realm_name }}</b>.{% endtrans %}
+{% endif %}
+
+{{ _('Organization URL:') }} {{ realm_name }} <br/>
+{{ _('Organization Type:') }} {{ organization_type }}<br/>
+{% if repository_link %} {{ _('Repository Link:') }} {{ repository_link }}<br/> {% endif %}
+{% if organization_nature %} {{ _('Organization Nature:') }} {{ organization_nature }}<br/> {% endif %}
+{% if organization_website %} {{ _('Organization website:') }} {{ organization_website }}<br/> {% endif %}
```

Comment:
Nitpick: if we're capitalizing the first letter, then we should ``Website`` as well